src/data.js

```diff
@@ -99,4 +99,13 @@ var items = [
     discord: "https://discord.gg/f2ApYdxm3h",
     logo: "https://fysix.xyz/static/logo.jpg"
   }
```

Comment:
please add this @xSkyyy ?

```suggestion
  },
  {
```